components/form/TimesheetEntryForm.tsx

```diff
@@ -122,8 +140,14 @@ function TimesheetEntryForm({
     }
 
     const handleSubmit = () => {
-        const entry = validateTimesheetEntryFields(timesheetEntryFields)
-        onSubmit(entry)
+        try {
+            const entry = validateTimesheetEntryFields(timesheetEntryFields)
+            onSubmit(entry)
+        } catch (error) {
```

Comment:
Is it possible that `catch` catches something else than an actual error? Could this be typed then?

Reply:
No. It catches errors that have been thrown in validateTimesheetEntryFields function. JS `try...catch` is just shit and you have to validate the error/exception type by yourself inside the catch. It's even in the [try...catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#conditional_catch-blocks) documentations. :/